python/dgl/graphbolt/base.py

```diff
@@ -108,22 +108,38 @@ class CopyTo(IterDataPipe):
 
     Functional name: :obj:`copy_to`.
 
-    This is equivalent to
+    When `data` has `to` method implemented, `CopyTo` will be equivalent to
 
     .. code:: python
 
        for data in datapipe:
            yield data.to(device)
 
+    For `gb.MiniBatch`, only a part of attributes will be transferred to
```

Comment:
`:class:~dgl.graphbolt.MiniBatch`

Reply:
refer to https://github.com/dmlc/dgl/blob/557a8f817a285acc9dfee91c19a19ff8728a8d76/python/dgl/dataloading/dataloader.py#L789